projects/components/src/lib/inline-message/inline-message.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="wrapper">
     <button type="button" (click)="openDialog()" class="inline-message-btn">i</button>
-    <div class="title">
-        <ng-content></ng-content>
+    <div class="title" *ngIf="title">
+        <ng-content *ngTemplateOutlet="title"></ng-content>
```

Comment:
If we want a title input, it probably belongs in the dialog component instead.  And it could just be a string instead of a TemplateRef.  As-is here, it's a div after the `i` button, and more like a duplicate of the field-label

Reply:
removed :)